<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildingInfoWidget</class>
 <widget class="QWidget" name="BuildingInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BuildingInfoWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="buildingNameLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="buildingOwnerLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="MyPushButton" name="buyOrSellpushButton">
       <property name="text">
        <string>Buy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="detailsPushButton">
       <property name="text">
        <string>See More Details</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">mypushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
